tools/pot/setup.py

```diff
@@ -70,7 +70,7 @@ def get_version():
     "scipy>=1.8; python_version >= '3.8'",
     "jstyleson>=0.0.2",
     "addict>=2.4.0",
-    "networkx<=2.8.8",
+    "networkx<=3.0",
```

Comment:
firstly,  `3.0` is not aligned with MO; also, `3.0` looks not safe for MO. Need more testing and to see how the API is changed.